src/helpers/datetime.js

```diff
@@ -461,10 +461,16 @@ export function convertPHPToMomentFormat(format) {
 		r: '', // no equivalent
 		U: 'X',
 	};
-
 	return String(format)
 		.split('')
-		.map((chr) => (chr in replacements ? replacements[chr] : chr))
+		.map((chr, index, elements) => {
+			// Allow the format string to contain escaped chars, like ES or DE needs
+			const last = elements[index - 1];
+			if (chr in replacements && last !== '\\') {
+				return replacements[chr];
+			}
+			return chr;
+		})
```

Comment:
@carstingaxion can we add a unit test for this? You can find them [here](https://github.com/GatherPress/gatherpress/blob/main/test/unit/js/src/helpers/datetime.test.js#L276). Thanks!!

Reply:
Please have a look, at my added test.